config/software/datadog-agent.rb

```diff
@@ -46,10 +46,28 @@
         copy 'packaging/debian/start_agent.sh', '/opt/datadog-agent/bin/start_agent.sh'
         command 'chmod 755 /opt/datadog-agent/bin/start_agent.sh'
       end
-      copy 'packaging/supervisor.conf', '/etc/dd-agent/supervisor.conf'
-      copy 'datadog.conf.example', '/etc/dd-agent/datadog.conf.example'
-      copy 'conf.d', '/etc/dd-agent/'
-      mkdir '/etc/dd-agent/checks.d/'
+
+      # XXX: we want 'arista' here but since i can't get that yet. im assuming
+      # here that rhel at Fastly is really meant for Arista/Fedora.
+      if ohai['platform_family'] == 'rhel'
+        command 'mkdir -p /etc/dd-agent/checks.d/'
+        command 'mkdir -p /etc/dd-agent/conf.d/'
+        command 'mkdir -p /persist/sys/dd-agent/checks.d/'
```

Comment:
As discussed prior, I think we can have the arista-metrics package set things up instead.